site/samples/jdbc.md

```diff
@@ -1,13 +1,8 @@
 ## JDBC Sample
 
-This sample WDT domain model section includes configuration for two JDBC data sources.
+These WDT domain model sample sections include configuration for JDBC data sources.
  
-The data source `datasource-1` has a sparse configuration meant to set a minimal number of values. It is not necessary to include additional attributes and sub-folders, unless you intend to override the default values.
-
-The data source `datasource-2` includes additional folders and attributes to provide further customization, including targeting information, connection pool configuration, and such. The cluster `cluster-1` should be defined elsewhere within this model, or should already exist in a domain that is being updated.  
-
-It is recommended that credential fields, such as `PasswordEncrypted`, should not be stored as clear text in the model. Those values can be referenced in a separate variables file or Kubernetes secret, or the model can be encrypted using the [Encrypt Model Tool](../encrypt.md).
-
+The first sample has a JDBC data source with a sparse configuration meant to set a minimal number of values. It is not necessary to include additional attributes and sub-folders, unless you intend to override the default values.
 ```yaml
 resources:
```

Comment:
Thjis sample needs properties that are normally updated by customers
JDBCConnectionPoolParams:
                    StatementCacheSize: 10
                    InitialCapacity: 0
                    MinCapacity: 0
                    TestTableName: SQL ISVALID
                    StatementCacheType: LRU
                    MaxCapacity: 5
                    ConnectionReserveTimeoutSeconds: 10
                JDBCDataSourceParams:
                    GlobalTransactionsProtocol: OnePhaseCommit
                    JNDIName: my.datasources.ds2

Reply:
That does not make sense @mriccell 
There are two examples here, the first, the minimum needed, and the second has the properties, no?

Reply:
I disagree, we should ask Steve Felts what are the most popular values that customers update.  I believe it is Max and Min number of connections, test connection on reserve, as well as properties dealing with the connections.

Reply:
You can remove 
JDBCDataSourceParams:
GlobalTransactionsProtocol: OnePhaseCommit
JNDIName: my.datasources.ds2

Reply:
I reduced this to just a single model, with all the fields we've discussed.